### PR TITLE
allow underscore in variable names

### LINKED ISSUE #7: Underscores not allowed in variable names
https://github.com/jbyuki/quickmath.nvim/blob/dcfc5450fa686714817a0d4767299f37f94bdb43/src/quickmath.lua.t#L57

As far as I can tell, lua's `%w` is unlike the "standard" [`\w`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Cheatsheet#character_classes) regex in that it does not include `_`. This means that variable names containing underscores are considered invalid (when they are valid in normal lua variable names, i.e. outside of quickmath).

### PR DESCRIPTION
**Pull request by hejops**
fixes https://github.com/jbyuki/quickmath.nvim/issues/7

### CODE REVIEW

File: lua/quickmath/init.lua [hunk 1]
```diff
@@ -24,7 +24,7 @@ local function StartSession()
 
 		for i, line in ipairs(content) do
 			local line = content[i]
-			if not string.find(line, "^%w+%s*=") and not string.find(line, "^%s*$") then
+			if not string.find(line, "^[A-Za-z0-9_]+%s*=") and not string.find(line, "^%s*$") then
```
Comment (jbyuki) [inline lua/quickmath/init.lua]: Wouldn't this match "1234"?

**Comment by jbyuki** [comment]
Thank you for the PR. 
Every place that needs change is covered I believe.

File: lua/quickmath/init.lua (hunk 1 above)
Comment (hejops) [reply lua/quickmath/init.lua]: Oh yeah, I forgot variables can't start with a digit, thanks for reminding me

**Comment by jbyuki** [comment]
Looks good. Thank you!

**Comment by hejops** [comment]
Happy to help!

**Pull request merged**